app/models/bus_stop.rb

```diff
@@ -10,4 +10,8 @@ def self.from_json(**params)
     BusStop.where(stop_id: stop_id, direction: direction)
            .first_or_create(stop_id: stop_id, name: name, direction: direction)
   end
+
+  def display_name
```

Comment:
I'm thinking we add something like ActiveModel Serializers, since these methods can't be accessed in the JSON.

Reply:
Not sure if this is the right git branch of activemodel serializers https://github.com/jtrudell/angry_transit_tracker/pull/2/commits/2e95d5ecc2c2fc459fc9fd38e33bab41e97596a9

Reply:
Done.